src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs

```diff
@@ -117,11 +113,7 @@ static Runspace CreateRunspace(RunspaceConfiguration runspaceConfiguration)
         /// <exception cref="ArgumentNullException">
         /// Thrown when runspaceConfiguration is null
         /// </exception>
-#if CORECLR
         internal
-#else
-        public
-#endif
         static Runspace CreateRunspace(PSHost host, RunspaceConfiguration runspaceConfiguration)
```

Comment:
For these 2 changes, please move `internal` to the same line of the rest of the method signature. Like:
`internal static Runspace CreateRunspace(RunspaceConfiguration runspaceConfiguration)`